boards/arm/mimxrt1015_evk/mimxrt1015_evk.dts

```diff
@@ -53,6 +53,7 @@ arduino_serial: &uart4 {};
 	reg = <0x402a8000 0x4000>, <0x60000000 0x1000000>;
 	at25sf128a: at25sf128a@0 {
 		compatible = "adesto,at25sf128a", "jedec,spi-nor";
+		size = <134217728>;
```

Comment:
its a nit, but for readability can we use DT_SIZE_M

Reply:
Could be confusing, since the size is measured in bits.  This one would have to be `DT_SIZE_M(128 * 8)`.  Also DT_SIZE_M appears to be Zephyr-specific, which isn't ideal.

Do you want to require this?  For consistency we should also change the existing `size` properties in other devicetree files.

Reply:
Ah, didn't realize size was in bits... (just curious, is it ever not a multiple of 8?).

Reply:
It's never not a multiple of 4096 * 8 since the sector size is always 4096 bytes.  Almost certainly never not a multiple of 65536 * 8, which is the standard block erase size (some chips support 32768 byte block erases).

In early days I used the hex representation, but switched to decimal because it was what most other people were using, and because it's easier for a human to recognize the size.  Few people can look at 0x8000000 and know immediately that it's a 128 Mbit (16 MByte) flash.

Reply:
Also, to look back at history: when I converted to DTS I made the size bits instead of bytes because the datasheets always specify the size in bits rather than bytes.  It's common in some applications to use multiple flash devices in parallel: bits makes more sense in that situation.

Reply:
Can we do something like:

`size = <134217728>; /* in bits */`

Reply:
Of course we could.  But why?  The property description in the binding specifies that the value is in bits.  A wrong value will be discovered quite rapidly.

This sort of thing is a slippery slope where for consistency we'd end up going through and adding a units value to every numeric device tree property that has units (timeout, other sizes, ...).  I don't really think that's a bad idea, but I also don't think it should be done ad hoc.